lib/jettywrapper.rb

```diff
@@ -42,7 +42,7 @@ class << self
     attr_writer :hydra_jetty_version, :url, :tmp_dir, :jetty_dir, :env
 
     def hydra_jetty_version
-      @hydra_jetty_version ||= 'v8.1.1'
+      @hydra_jetty_version ||= 'v8.2.0'
```

Comment:
@cbeer well, it doesn't. Was introduced here: https://github.com/projecthydra/jettywrapper/commit/d325400a413cf3ee6377e6b40acad2ce698e2864 But if you want to remove it it'll have to go into version 3.0.0